File: Backend/Services/ViewModels/Company/AllCompanyDataModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Services.ViewModels.Company
{
  public  class AllCompanyDataModel
    {
        public int companyID { get; set; }
        public double overAllRate  { get; set; }
        public string companyName { get; set; }
        public string categoryName { get; set; }
        public string  img{ get; set; }
        public int Category_Id { set; get; }
        public string City_Name { get; set; }

    }
}

File: Mudita Frontend/MuditaFrontend/src/app/_models/company/all-company-data-model.ts
import { Category } from 'src/app/_models/Category/category';

export class AllCompanyDataModel {
    constructor(
        public companyID?:number,
        public overAllRate?:number,
        public companyName?:string,
        public categoryName?:string,
        public img?:string,
        public city_Name?:string,
    ){}
}

export class companyModel{
    constructor(
        public managerID?:string,
        public companyName?:string,
        public phone?:number,
        public email?:string,
        public fax?: number,
        public website?: string,
        public description?: string,
        public category?:string,
        public categoryID?:string,
        public foundedDate?:Date,
        public manager?:string,
        public citis?:string[],
        public linkedProfile?:string,
        public faceProfile?:string,
        public citisID?:number[],
        public countries?:string[],
        public countriesID?:number[],
        public Reiews?:string[],
        public overAllRate?:number,
        public img?:string
    ){}
}


